build_integration_test.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+cli="./faas-cli"
+template="python3"
+if [ $(uname) == 'Darwin' ] ; then
+    cli="./faas-cli-darwin"
+elif [ -f "/etc/debian_version" ] ; then
+    if [ $(dpkg --print-architecture) == 'armhf' ] ; then
+        cli="./faas-cli-armhf"
+        template="python3-armhf"
+    fi
+fi
+
+build_faas_function() {
+
+    function_name=$1
+
+    eval $cli new $function_name --lang $template
+
+cat << EOF > $function_name/handler.py
+def handle(req):
+
+    return "Function output from integration testing: Hello World!"
+EOF
+
+    eval $cli build -f $function_name.yml
+}
+
+run_and_test() {
+    function_name=$1
+    port=$2
+    timeout=$3
+
+    if lsof -Pi :$port -sTCP:LISTEN -t >/dev/null ; then
+        echo "Port $port is already allocated.. Cannot use this port for testing.
+        Exiting test..."
+
+        echo "Removing function image $function_name:latest"
+        docker rmi -f $function_name:latest
+        exit 1
+    fi
+
+    id=$(docker run --env fprocess="python3 index.py" --name $function_name -p $port:8080 -d $function_name:latest)
+
+    function_up=false
+    for (( i=1; i<=$timeout; i++ ))
```

Comment:
Does this work in place?

```
for i in {1...$timeout}
```

Reply:
It's not working with variable, referring to the documentation and it failed on practice as well